<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MedicalRecord</class>
 <widget class="QMainWindow" name="MedicalRecord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTextEdit" name="text_utter"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTextEdit" name="text_utter_one">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="last_utter">
            <property name="text">
             <string>上一句</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="next_utter">
            <property name="text">
             <string>下一句</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="tagger_board">
          <item>
           <widget class="QPushButton" name="chief_complain">
            <property name="text">
             <string>主诉(0)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="now_history">
            <property name="text">
             <string>现病史(1)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="past_history">
            <property name="text">
             <string>既往史(2)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="physical_exam">
            <property name="text">
             <string>体检(3)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="aux_inspect_rst">
            <property name="text">
             <string>辅助检查结果(4)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="preli_diag">
            <property name="text">
             <string>初步诊断(5)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="treat_opinion">
            <property name="text">
             <string>治疗意见(6)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="other">
            <property name="text">
             <string>其它(7)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="button_openFile">
          <property name="text">
           <string>打开文件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="text">
           <string>保存路径</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="text_numOfAll"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="button_lastLine">
        <property name="text">
         <string>上一条</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_nextLine">
        <property name="text">
         <string>下一条</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_save">
        <property name="text">
         <string>保存</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_exit">
        <property name="text">
         <string>退出</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
